include/RED4ext/Scripting/Stack.hpp

```diff
@@ -31,20 +31,24 @@ using StackArgs_t = std::vector<CStackType>;
 struct IStack
 {
     // clang-format off
-    virtual ~IStack() = default;                            // 00
-    virtual void* GetResultAddr() { return nullptr; };      // 08
-    virtual CBaseRTTIType* GetType() { return nullptr; };   // 10
-    virtual void sub_18(int64_t a2){};                      // 18
+    virtual ~IStack() = default;                          // 00
```

Comment:
Is this intended? I remember seeing a vtbl with 4 vfuncs returning default values and the being overridden with a real one, might be a `CSomeStack : CBaseStack` and ctor inlined?

Reply:
It isn't no :o